src/views/Players/EditPlayer.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react'
+import { updatePlayerById } from '../../services/players';
+import PlayerForm from '../../components/PlayerForm';
+import { useParams } from 'react-router';
+import { useHistory } from 'react-router';
+
+export default function EditPlayer() {
+    const {id} = useParams();
+    const history = useHistory()
+    const [name, setName] = useState('')
+    const [position, setPosition] = useState('');
+    const [teamId, setTeamId] = useState('');
+   
```

Comment:
If you want to preload the inputs with the player's original info you can use a useEffect to get the player by id, and then set the name, position, and teamId to state. The inputs in the form would need value changed to defaultValue to be able to edit them.